app/views/idv/doc_auth/verify.html.erb

```diff
@@ -1,21 +0,0 @@
-<div id="form-steps-wait-alert"></div>
-
-<% if flow_session[:had_barcode_read_failure] %>
-  <%= render AlertComponent.new(
-        type: :warning,
-        class: 'margin-bottom-4',
-        text_tag: 'div',
-      ) do %>
-    <%= t(
-          'doc_auth.headings.capture_scan_warning_html',
-          link: render(
-            FormLinkComponent.new(
-              href: idv_doc_auth_step_path(step: :redo_document_capture),
-              method: :put,
-            ).with_content(t('doc_auth.headings.capture_scan_warning_link')),
-          ),
-        ) %>
-  <% end %>
-<% end %>
-
-<%= render 'idv/shared/verify', pii: pii, step_url: step_url, remote_identity_proofing: true %>
```

Comment:
Should we remove `remote_identity_proofing` handling from `app/views/idv/shared/_verify.html.erb` / `app/views/idv/in_person/verify.html.erb`?

Reply:
We can, that is probably a different PR however. Additionally, we'll have to see what ends up happening to that template overall since, ideally, in-person will adopt the pattern here.